packages/bridge/src/vl/time-series/README.md

```diff
@@ -70,6 +71,9 @@ BucketRanges = [number, number]
 `readOnly` is a boolean to specify whether this widget should filter or not.
 `nBuckets` is a number to specify the number of buckets the histogram should have
 `bucketRanges` is an array of pairs of numbers, which lets you specify the bucket ranges by hand
+`duration` how long should the animation be, defaults to 30
```

Comment:
It would be nice to add the units, which are 'seconds'

Reply:
:+1: Done in another branch, I've also changed it to 10, which is VL's default